types/mydecimal_test.go

```diff
@@ -145,6 +145,41 @@ func (s *testMyDecimalSuite) TestToFloat(c *C) {
 	}
 }
 
+func (s *testMyDecimalSuite) TestToHashKey(c *C) {
```

Comment:
This is not a standard format of unit test in golang.
You can check the format of other test functions in this file.

Reply:
Emm, which part is not the standard format?